pkgs/top-level/all-packages.nix

```diff
@@ -1676,6 +1676,8 @@ with pkgs;
 
   sqlint = callPackage ../development/tools/sqlint { };
 
+  sqlcheck = callPackage ../development/tools/sqlcheck { };
```

Comment:
It should be further down in the `### DEVELOPMENT / TOOLS` section, right before `sqlitebrowser`

Also your default.nix could live under `development/tools/database/sqlcheck/` bat that's a minor nitpick.

Reply:
sure, updated